vic/drivers/shared_image/src/vic_store.c

```diff
@@ -157,7 +173,8 @@ vic_store(dmy_struct *dmy_current,
                         v = veg_con_map[i].vidx[m];
                         if (v >= 0) {
                             dvar[i] =
-                                (double) all_vars[i].cell[v][k].layer[j].ice[p];
+                                (double) all_vars[i].cell[v][k].layer[j].ice
+                                [p];
```

Comment:
Even if `uncrustify` enforced this, this is an odd break. Better to break after the `(double)`